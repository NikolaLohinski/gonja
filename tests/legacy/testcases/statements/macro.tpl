{% macro input(name, value='', type='text', size=20) -%}
    <input type="{{ type }}" name="{{ name }}" value="{{
        value|e }}" size="{{ size }}">
{%- endmacro -%}
<p>{{ input('username') }}</p>
<p>{{ input('username', type='password') }}</p>
<p>{{ input(name='address', size=42) }}</p>
