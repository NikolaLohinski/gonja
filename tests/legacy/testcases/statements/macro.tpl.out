<p><input type="text" name="username" value="" size="20"></p>
<p><input type="password" name="username" value="" size="20"></p>
<p><input type="text" name="address" value="" size="42"></p>
